server/templates/base.html

```diff
@@ -132,7 +132,7 @@ <h6>Data Commons</h6>
               <a href="#">About Data Commons</a>
               <a href="{{ url_for('static.faq') }}">Frequently Asked Questions</a>
               <a href="http://github.com/datacommonsorg">Github Repository</a>
-              <a href="{{url_for('sitemap.index')}}">Sitemap</a>
+              <a href="{{url_for('placelist.index')}}">Place List</a>
```

Comment:
Could you remove this link actually? I think we wanted to both rename and remove it from this list.

Reply:
Done